internal/pkg/tui/installer/state.go

```diff
@@ -80,38 +81,40 @@ func NewState(ctx context.Context, conn *Connection) (*State, error) {
 		conn: conn,
 		opts: opts,
 		pages: []*Page{
-			NewPage("Machine Config",
+			NewPage("Installer Params",
 				NewItem(
-					"cluster name",
-					v1alpha1.ClusterConfigDoc.Describe("clusterName", true),
-					&opts.ClusterConfig.Name,
+					"image",
+					v1alpha1.InstallConfigDoc.Describe("image", true),
+					&opts.MachineConfig.InstallConfig.InstallImage,
 				),
 				NewItem(
-					"control plane endpoint",
-					v1alpha1.ControlPlaneConfigDoc.Describe("endpoint", true),
-					&opts.ClusterConfig.ControlPlane.Endpoint,
+					"install disk",
+					v1alpha1.InstallConfigDoc.Describe("disk", true),
+					&opts.MachineConfig.InstallConfig.InstallDisk,
+					diskInstallOptions...,
 				),
+			),
+			NewPage("Machine Config",
 				NewItem(
 					"machine type",
 					v1alpha1.MachineConfigDoc.Describe("type", true),
 					&opts.MachineConfig.Type,
 					machineTypes...,
 				),
 				NewItem(
-					"kubernetes version",
-					"Kubernetes version to install.",
-					&opts.MachineConfig.KubernetesVersion,
+					"cluster name",
+					v1alpha1.ClusterConfigDoc.Describe("clusterName", true),
+					&opts.ClusterConfig.Name,
 				),
 				NewItem(
-					"install disk",
-					v1alpha1.InstallConfigDoc.Describe("disk", true),
-					&opts.MachineConfig.InstallConfig.InstallDisk,
-					diskInstallOptions...,
+					"control plane endpoint",
+					v1alpha1.ControlPlaneConfigDoc.Describe("endpoint", true),
+					&opts.ClusterConfig.ControlPlane.Endpoint,
 				),
 				NewItem(
-					"image",
-					v1alpha1.InstallConfigDoc.Describe("image", true),
-					&opts.MachineConfig.InstallConfig.InstallImage,
+					"kubernetes version",
+					"Kubernetes version to install.",
```

Comment:
Are we missing the auto doc here?

Reply:
This kubernetes version actually does not have one. We have doc string for the image, but not for the image version.

Reply:
I believe we don't have that as a single config option? we have multiple which control the k8s version

Reply:
yeah, this kubernetes version is passed into generate package, then it's making some magic inside. We don't have 1:1 mapping for it in the config.